plugins/cost-insights/src/api/CostInsightsApi.ts

```diff
@@ -112,12 +112,14 @@ export type CostInsightsApi = {
    * @param group
    * @param duration A time duration, such as P1M. See the Duration type for a detailed explanation
    * of how the durations are interpreted in Cost Insights.
+   * @param lastBillingDate The most current date for which billing data is complete, in YYYY-MM-DD format.
    * @param project (optional) The project id from getGroupProjects or query parameters
    */
   getProductInsights(
     product: string,
     group: string,
     duration: Duration,
+    lastBillingDate: string,
     project: Maybe<string>,
```

Comment:
Params for this method are starting to get a bit verbose :/

Reply:
In general with more than like 1-2 parameters, we have started favoring passing in a single object with the corresponding fields instead - long lists of unnamed (at the call site) params are a huge pain